### ISSUE TITLE
Setting to adjust font size on Android

### ISSUE THREAD

**Issue by surfdude29** [bluesky-social/social-app#3197]
**Is your feature request related to a problem? Please describe.**

Some users on Android would like the ability to adjust font size for the Bluesky app independently of their system font size setting. (This is possible on iOS but isn't currently available on Android.)

**Describe the solution you'd like**

Please allow either all users or at least only Android users to change the font size setting for the app with a setting inside the app.

**Describe alternatives you've considered**

Ideally, if Android allowed users to customise font size settings per app, this feature request would be unnecessary.

Alternatively, users can adjust their system font setting and simply live with how the Bluesky app looks using those settings.

**Additional context**

This issue is inspired by a conversation with an Android user who requested this (but I didn't check the wording with them so I don't want to imply they necessarily endorse this):

https://bsky.app/profile/fukang.bsky.social/post/3knkgle3yxs22

**Comment by abhijithgopan** [comment]
This would be nice, the Bluesky font size is much larger for me too compared to other apps.

**Comment by lovelydumpling** [comment]
Yeah I'm having a similar issue on a tablet, where the text is kind of too small but I can't increase it :(

**Comment by mozzius** [comment]
Done

**Comment by onelittlehope** [comment]
@mozzius Under `Settings > Appearance > Font size`, would it be possible to add additional sizes for people who are near sighted? My current workaround is to use larger system font sizes but this breaks the UI of several other apps and so I have to switch back and forth.
